src/components/IDForm/IDForm.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import IDForm from './IDForm.js';
+import { render, screen } from '@testing-library/react';
+
+
+describe('IDForm', () => {
```

Comment:
Nice job with this, I wasn't sure if we should test the whole thing or only one piece to begin with. If we test that one piece shows up then we can imply it all shows up I think. However it feels safer testing that all the pieces of the component render!

Reply:
I agree with taylor. It's like with Unit testing and integration testing.